sql/moz-fx-data-shared-prod/org_mozilla_mozregression_derived/mozregression_aggregates_wchan1_v1/schema.yaml

```diff
@@ -0,0 +1,25 @@
+fields:
```

Comment:
Could you add field descriptions to `mozregression_variant` for example? I don't know what that means or at the very least a link to docs?

Reply:
This is an onboarding exercise to try creating a derived dataset.  No one will be using this dataset and it will be removed.